Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1160,6 +1160,15 @@ ExceptionOr<void> HTMLElement::showPopoverInternal(const HTMLElement* invoker)
     return { };
 }
 
+void HTMLElement::setInvoker(HTMLElement* invoker)
+{
+    if (RefPtr oldInvoker = popoverData()->invoker(); oldInvoker && !invoker)
```

Comment:
I'm not sure I understand why `!invoker` is correct here. If it is, please add a comment, if not, please remove.